<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoutingWidget</class>
 <widget class="QWidget" name="RoutingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>414</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="routingLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="addViaButton">
       <property name="text">
        <string>Add Via</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reverseRouteButton">
       <property name="text">
        <string>Reverse</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearRouteButton">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="routingProfileComboBox">
       <item>
        <property name="text">
         <string>Car (fastest)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Car (shortest)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bicycle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pedestrian</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="configureButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../marble.qrc">
         <normaloff>:/icons/16x16/configure.png</normaloff>:/icons/16x16/configure.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>No results.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="showInstructionsButton">
       <property name="text">
        <string>Show Details</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="directionsListView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="routeComboBox">
       <property name="toolTip">
        <string>Choose alternative routes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openRouteButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../marble.qrc">
         <normaloff>:/icons/16x16/document-open.png</normaloff>:/icons/16x16/document-open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveRouteButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../marble.qrc">
         <normaloff>:/icons/16x16/document-save.png</normaloff>:/icons/16x16/document-save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../marble.qrc"/>
 </resources>
 <connections/>
</ui>
